android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/Account.kt

```diff
@@ -0,0 +1,40 @@
+package com.mytiki.sdk.capture.receipt.capacitor
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import com.getcapacitor.JSObject
+import com.microblink.digital.PasswordCredentials
+import com.mytiki.sdk.capture.receipt.capacitor.req.ReqAccount
+import com.mytiki.sdk.capture.receipt.capacitor.rsp.RspAccount
+import com.mytiki.sdk.capture.receipt.capacitor.rsp.RspAccountList
+
+class Account(
+    val accountCommon: AccountCommon,
+    val username: String,
+    val password: String? = null,
+    var isVerified: Boolean? = null
+) {
+    fun toRsp(): JSObject = JSObject.fromJSONObject(RspAccount(this).toJson())
+    companion object{
```

Comment:
companion object should be in the top

Reply:
sorry, my mistake... it should be in the bottom actually
https://kotlinlang.org/docs/coding-conventions.html#class-layout